lib/linter-views.coffee

```diff
@@ -124,8 +123,9 @@ class LinterViews
     @markers = []
 
   removeBubble: ->
-    @bubble?.destroy()
-    @bubble = null
+    # @bubble?.destroy()
+    # @bubble = null
+    @renderedMessages.map (msg) -> msg.destroy()
```

Comment:
That's a mistake, it should be pushed in the renderBubble function as the messages are created